code/modules/vending/clothesmate.dm

```diff
@@ -71,7 +71,8 @@
 		            /obj/item/clothing/under/scratch = 2,
 		            /obj/item/clothing/under/gimmick/rank/captain/suit/skirt = 2,
 		            /obj/item/clothing/under/gimmick/rank/head_of_personnel/suit/skirt = 2,
-		            /obj/item/clothing/suit/jacket = 2,
+		            /obj/item/clothing/suit/hooded/raincoat = 2,
+					/obj/item/clothing/suit/jacket = 2,
```

Comment:
needs to be aligned

Reply:
![image](https://user-images.githubusercontent.com/46236974/159361657-b600a646-74de-4f7c-96fa-062d020b6349.png)
![image](https://user-images.githubusercontent.com/46236974/159361296-f5bdf033-f068-437d-8dd5-d5a60841589a.png)
![image](https://user-images.githubusercontent.com/46236974/159361350-ab9473e6-4fc4-4ae9-90ea-39d838e547e9.png)

uhhhhhhh is something broken because it looks aligned everywhere i see it?